doc/usage/integration.rst

```diff
@@ -8,9 +8,41 @@
 Software library
 ================
 
-The most stable part of Celeritas is, at the present time, the high-level
-:ref:`api_g4_interface`. However, many other
-components of the API are stable and documented in the :code:`api` section.
+Using Celeritas in a downstream Geant4 application is meant to be as easy
+possible.
+
+1. Find and link in your CMake project:
+
+   - Find the Celeritas package.
+   - Link ``Celeritas::accel`` .
+   - Use ``celeritas_target_link_libraries`` instead of
+     ``target_link_libraries`` when both VecGeom and CUDA are enabled.
+
+2. Set up physics offloading in your "main" function.
+
+   - Register the ``TrackingManagerConstructor``, which tells Geant4 to send EM
+     tracks to Celeritas rather than the main tracking loop.
+   - Tweak the ``SetupOptions`` based on problem requirements, or use the
+     :ref:`g4_ui_macros`.
+
+3. Add hooks to safely set up and tear down Celeritas and its GPU code.
+
+   - Initialize logging in ``UserActionInitialization``, using
+     ``BuildForMaster`` (MT mode) or ``Build`` (serial).
+   - In ``UserRunAction``, ``BeginOfRunAction`` initializes problem data
+     (master or serial) and local state (worker or serial), and
+     ``EndOfRunAction`` safely deallocates everything.
+
+The changes for a simple application look like:
+
+.. literalinclude:: ../../example/accel/add-celer.diff
+   :language: diff
+
+More details of integration are described below.
+
+The most stable part of Celeritas is the high-level :ref:`api_g4_interface`.
+Other components of the API are stable and documented in the :code:`api`
+section.
```

Comment:
Did you mean something like

```suggestion
The most stable part of Celeritas is the high-level :ref:`api_g4_interface`,
with the API components documented in the :code:`api` section.
```

Reply:
I'll try to clarify a bit more. The geant4 interface is very stable; other components are mostly stable...